buildspec/linuxIntegrationTests.yml

```diff
@@ -15,13 +15,13 @@ phases:
       - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay&
       - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
       - apt-get update
-      - apt-get install -y jq python2.7 python3.6 python3.7 msbuild
+      - apt-get install -y jq python2.7 python-pip python3.6 python3.7 msbuild
```

Comment:
why are all of these versions installed? Which one is `pip3` supposed to serve?

Reply:
We test all the supported Lambda runtimes can build Lambdas and execute debuggers against them. Due to ABI issues, the runtime and build pythons have to be same version